### ISSUE TITLE
AssertArrayEquals - show a more useful error message when failing

### ISSUE THREAD

**Issue by dylan-chong** [junit-team/junit4#1471]
With the code

```
assertArrayEquals(new int[]{1,2,3}, new int[]{2,3,4,5});
```

I get the error message

```
java.lang.AssertionError: array lengths differed, expected.length=3 actual.length=4
```

But this doesn't tell me much about why it failed - if the length is wrong, then what are the extra values?

I would rather see an error message that shows both array contents:

```
java.lang.AssertionError: 
Expected :[1, 2, 3]
Actual   :[2, 3, 4, 5]
```

Which is actually what you get when you convert your arrays to lists first.

```
assertEquals(Arrays.asList(1,2,3), Arrays.asList(2,3,4,5));
```

Could the output format be changed to show both the contents of both arrays?

**Comment by kcooney** [comment]
It looks like this was fixed by #1300

@dylan-chong can you confirm that the above pull fixes your issue?

**Comment by dylan-chong** [comment]
Ah yes that looks like it fixes this issue. Thanks.
